src/main/app/src/app/dashboard/dashboard.component.ts

```diff
@@ -17,13 +17,16 @@ import {CdkDragDrop, moveItemInArray, transferArrayItem} from '@angular/cdk/drag
 import {GebruikersvoorkeurenService} from '../gebruikersvoorkeuren/gebruikersvoorkeuren.service';
 import {forkJoin} from 'rxjs';
 import {DashboardCardInstelling} from './model/dashboard-card-instelling';
+import {MatMenuTrigger} from '@angular/material/menu';
 
 @Component({
     templateUrl: './dashboard.component.html',
     styleUrls: ['./dashboard.component.less']
 })
 export class DashboardComponent implements OnInit {
```

Comment:
Ik heb alsnog het idee dat het een stuk simpeler opgezet kan worden als je een enkele array gebruikt icm de juiste flexlayout. Dan hoef je ook alleen maar de volgorde te onthouden + wel of niet tonen.

Reply:
Aan de andere kant zie ik ook dat angular zelf ook bezig is om dit fatsoenlijk te gaan ondersteunen: https://angular.io/guide/roadmap#support-two-dimensional-drag-and-drop.

Wellicht is het momenteel niet gemakkelijker te krijgen